index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,500|Nunito&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>HTML Form</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav class="navbar">
+        <div class="nav-desc">
+          <p>Already have an Intuit account?</p>
+          <p><a href="#">Forgot my User ID or Password?</a></p>
+        </div>
+        <button>
+          <span> <i class="fa fa-lock" aria-hidden="true"></i></span> Sign In
+        </button>
+      </nav>
+      <div class="container-content">
+        <div class="inner">
+          <header class="header">
+            <div class="main-icon">
+              <img src="images/main-logo.png" alt="Main logo" />
+            </div>
+            <div class="icons-set">
+              <span> <img src="images/mint-logo.png" alt="" /> mint</span>
+              <span>
+                <img src="images/quickbooks-logo.png" alt="" /> quickbooks</span
+              >
+              <span>
+                <img src="images/turbotax-logo.png" alt="" /> turbotax</span
+              >
+            </div>
+          </header>
+          <section>
+            <h2>Create an Intuit account</h2>
+            <div class="header-closure">
+              <p>One account for everything Intuit, including Mint.</p>
+              <p><a href="#">Learn more</a></p>
+            </div>
+            <form class="form" action="#" method="post">
+              <fieldset class="inputs">
+                <label for="">
```

Comment:
Let's give `<input type="email" name="email" />` an `id` attribute, then use it as a value for the `for` attribute here.
```suggestion
                <label for="<id of email input>">
```
See detailed explanation above.